checkov/cloudformation/cfn_utils.py

```diff
@@ -2,12 +2,17 @@
 import os
 from typing import Optional, List
 
+import dpath.util
+
 from checkov.cloudformation.checks.resource.registry import cfn_registry
+from checkov.cloudformation.context_parser import ContextParser
 from checkov.cloudformation.parser import parse
-from checkov.cloudformation.parser.node import dict_node, list_node
+from checkov.cloudformation.parser.node import dict_node, list_node, str_node
 from checkov.common.runners.base_runner import filter_ignored_paths
 
-CF_POSSIBLE_ENDINGS = [".yml", ".yaml", ".json", ".template"]
+CF_POSSIBLE_ENDINGS = frozenset([".yml", ".yaml", ".json", ".template"])
+CF_DEFINITION_TYPES = frozenset(['METADATA', 'PARAMETERS', 'MAPPINGS', 'CONDITIONS', 'RESOURCES', 'OUTPUTS'])
```

Comment:
@Saarett Can we leverage  the `CloudformationTemplateSections` in https://github.com/bridgecrewio/checkov/blob/master/checkov/cloudformation/graph_builder/graph_components/block_types.py ?